src/bicep/modules/privateLink.bicep

```diff
@@ -0,0 +1,211 @@
+@description('The name of the resource the private endpoint is being created for')
```

Comment:
nit: style

For all of the things in this module let's be consistent with the rest of the codebase: variables, parameters, and resources are camelCased not underscore_separated